packages/headless/src/features/case-assist/case-assist-analytics-actions.ts

```diff
@@ -271,14 +296,26 @@ export const logDocumentSuggestionRating = (
         ticket: caseAssistCaseSelector(state),
       });
     },
-    analyticsType: 'caseAssist.documentSuggestionFeedback',
-    analyticsPayloadBuilder: (state): CaseAssist.DocumentSuggestionFeedback => {
-      return {
-        documentSuggestion: {
-          id: suggestionId,
-          responseId: state.documentSuggestion?.status?.lastResponseId || '',
-        },
-        liked: !!rating,
-      };
+    analyticsType: 'CaseAssist.DocumentSuggestionFeedback',
+    analyticsPayloadBuilder: (
+      state
+    ): CaseAssist.DocumentSuggestionFeedback | undefined => {
+      const responseId = state.documentSuggestion?.status?.lastResponseId;
+      const documentSuggestion = caseAssistDocumentSuggestionSelector(
+        state,
+        suggestionId
+      );
+      if (responseId) {
+        return {
+          responseId,
+          itemMetadata: {
+            uniqueFieldName: 'uniqueId',
+            uniqueFieldValue: documentSuggestion.suggestionId,
+            url: documentSuggestion.suggestion.documentUri,
+            title: documentSuggestion.suggestion.documentTitle,
+          },
+          liked: !!rating,
```

Comment:
That's actually a problem that we have between legacy UA and EP.
With the legacy UA the analytics action to rate a document suggestion accepts a number from 1 to 5 as we used have a stars rating system to rate document suggestion. However, with EP the analytics action instead of taking a number it takes a boolean for if the document suggestion is liked or not.

so we are planing to fix this in the next v4 where we will completely remove all what is legacy analytics: https://coveord.atlassian.net/browse/SFINT-5471